src/components/__internal__/no-choice-list-item.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ListOption from '../list-option/list-option.jsx';
+import styles from '../no-choice-list-item.css';
```

Comment:
The styles file for a component should live next to it and follow the same filename pattern. This ensures we're coupling a component to 1 stylesheet and it is always obvious where it should go. When the stylesheet composes from other stylesheets is where the magic happens for associating a component to stylesheets outside its folder.